static/coffee/controllers.coffee

```diff
@@ -270,23 +324,29 @@ controllers.controller('MessagesController', ['$scope', '$timeout', '$interval',
     )
 
   $scope.fetchOldItems = (search, startTime, page) ->
-    $scope.autoRefresh = ->
-      MessageService.fetchOld(search, startTime, page).then((data) ->
-        $scope.items = data.results
-        $scope.oldItemsMore = data.hasMore
-        $scope.oldItemsLoading = false
-      )
-    $scope.autoRefresh()
-    $interval.cancel($scope.autoRefresh)
-    $interval($scope.autoRefresh, 10000)
-    
-  $scope.$on '$destroy', ->
-    $interval.cancel($scope.autoRefresh)
+    return MessageService.fetchOld(search, startTime, page)
 
   $scope.onExpandMessage = (message) ->
     $scope.expandedMessageId = message.id
 
   #----------------------------------------------------------------------------
+  # Set busy state for individual messages when actioned before poll interval
+  #----------------------------------------------------------------------------
+  $scope.busy = (results, message) ->
+    busyMessage = []
+    for item in message
+      if item.id in results.messages
+        busyMessage.push(item.text)
+        item.busy = true
+        item.selected = false
+        $scope.expandedMessageId = false
+        $scope.updateItems()
+
+    # show busy alert
+    busyMessages = busyMessage.join('</li><li>')
+    UtilsService.displayAlert('error', '<strong>The following message(s) are busy:</strong><br><ul><li>' + busyMessages + '</ul>')
```

Comment:
I think we need a closing `</li>` before `</ul>` here.

Dynamically creating markup via strings isn't great, but I guess we don't have better options here. Would be cool if we were able to have a custom directive for this modal instead, but I guess that is quite a bit more work than it is worth.